crux-http-server/src/crux/http_server.clj

```diff
@@ -254,6 +254,10 @@
 (defn- attribute-stats [^ICruxAPI crux-node]
   (success-response (.attributeStats crux-node)))
 
+(defn- tx-committed? [^ICruxAPI crux-node request]
+  (let [submitted-tx (body->edn request)]
+    (success-response (.hasTxCommitted crux-node submitted-tx))))
```

Comment:
yep - if this throws the NodeOutOfSyncException, though, we'll want to return a 400 ('bad request') status (unless there's a better 4xx error?)